### PR TITLE
Feature#21 navigation

### PR DESCRIPTION
**Pull request by yovanha**
# Description
What did you implemented/Why did you implemented this:
 - We've created a button in the user Flight list that navigate the users to the "From" screen Why? That is the name of the first screen that users need to complete when they start to make a new reservation.
 - We've added a "logout" Icon in MyFlights screen that sends users to the LogIn screen once they sign in into the application.
 - We've added navigation in the buttons of the Log In and Sign Up screen to send the users to the application once they put correctly their credentials. 
   
  ## Screenshots 
image

### CODE REVIEW

File: FlightReservation/src/screens/FlightReservation.js [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+import PlusButton from './../components/flights/PlusButton';
 
-const flightList: JSX.Element[] = [];
-for (let i = 0; i < 8; i++) {
-  flightList.push(<FlightDestiny />);
-}
+// const flightList: JSX.Element[] = [];
+// for (let i = 0; i < 8; i++) {
+//   flightList.push(<FlightDestiny />);
+// }
```
Comment (DemisRincon) [inline FlightReservation/src/screens/FlightReservation.js:15]: remove commentaries

File: FlightReservation/src/screens/FlightReservation.js [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+      <ScrollView>
+        <View>
+          <ArrowBack
+            navigation={() => navigation.navigate('LogIn')}
+            name={'logout'}
+          />
+          <ScreenGreenTitle span={'flights'} />
+          <View style={style.inferior_content_container}>
+            <FlightDestiny />
+            {/* {flightList} */}
```
Comment (DemisRincon) [inline FlightReservation/src/screens/FlightReservation.js:28]: remove this

**Review by DemisRincon — CHANGES_REQUESTED** [review]

File: FlightReservation/src/screens/FlightReservation.js (hunk 1 above)
Comment (hancel-avila) [reply FlightReservation/src/screens/FlightReservation.js]: Done. 👍🏻

File: FlightReservation/src/screens/FlightReservation.js (hunk 2 above)
Comment (hancel-avila) [reply FlightReservation/src/screens/FlightReservation.js]: Done. 👍🏻

**Review by DemisRincon — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by cortezroberto — APPROVED** [review]
Well done

**Pull request merged**